core/modules/path/path.admin.inc

```diff
@@ -858,8 +852,7 @@ function path_bulk_update_batch_finished($success, $results, $operations) {
     }
   }
   else {
-    $error_operation = reset($operations);
-    backdrop_set_message(t('An error occurred while processing @operation with arguments : @args', array('@operation' => $error_operation[0], '@args' => print_r($error_operation[0], TRUE))));
+    backdrop_set_message(t('An error occurred while generating the URL aliases.'), 'error');
```

Comment:
I removed the printing of `$error_operation` because in our failures, that value was NULL and causing a PHP notice by trying to access it as an array.

My thinking is that if failures occurred, other PHP notices/warnings or watchdog calls would have happened where the database failed to write or an alias already existed, etc.

Reply:
> My thinking is that if failures occurred, other PHP notices/warnings or watchdog calls would have happened...

Makes sense. So let's stick with that simple message here. :+1: